scripts/build_utils.sh

```diff
@@ -1656,10 +1656,6 @@ function retrycmd_if_failure() {
 function set_centos_python3_version() {
     # This function expects $1 to be a string like "python3.9"
     local EXPECTED_PYTHON3_VERSION=$1
-    local EXPECTED_PYTHON3_VERSION_MASHED=$(echo $EXPECTED_PYTHON3_VERSION | tr -d '.')
-    for package in $(rpm -qa | grep -E '^python3[1-9]{1,3}' | grep -v $EXPECTED_PYTHON3_VERSION_MASHED); do
-        sudo dnf remove -y $package
-    done
     sudo dnf reinstall -y $EXPECTED_PYTHON3_VERSION || sudo dnf install -y $EXPECTED_PYTHON3_VERSION
-    sudo alternatives --auto python3
+    sudo ln -fs /usr/bin/$EXPECTED_PYTHON3_VERSION /usr/bin/python3
```

Comment:
Why not use `PATH` to do this?

Reply:
I'm not sure what you mean but I am open to alternatives (see what I did there)

Reply:
On second thought this is probably the cleanest way in this context